images/base/persistent_volume_scrubber.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Follow the pattern in our hack scripts - three explicit lines at the top, below the description of the script.